python/data_structures/stack.py

```diff
@@ -1,12 +1,32 @@
+from data_structures.linked_list import Node
+from data_structures.invalid_operation_error import InvalidOperationError
+
+
 class Stack:
     """
     Put docstring here
     """
 
-    def __init__(self):
-        # initialization here
-        pass
+    def __init__(self, top = None):
+        self.top = None
```

Comment:
Again, accepting an argument and not using it. Did you mean to write `self.top = top` on line 11?